indico/modules/events/editing/client/js/components/Timeline/ReviewForm.jsx

```diff
@@ -43,6 +46,10 @@ const judgmentOptions = [
 ];
 
 export default function ReviewForm({block}) {
+  const dispatch = useDispatch();
+  const {contributionId, eventId, editableType} = useSelector(getStaticData);
+  const {editors} = useSelector(getDetails);
+  const isEditor = editors.find(editor => editor.id === Indico.User.id) !== undefined;
```

Comment:
when would you have more than one editor on a revision?

also, i would send explicit permissions back from the server